docs/UserGuide.md

```diff
@@ -239,6 +239,22 @@ Examples:
 * `country note c/CN nt/building good relations (guanxi) is important when conducting business here`
 * `country note c/IN nt/is world's fastest growing economy`
 
+### Getting suggestions on clients: `suggest`
+
+Obtains a list of clients based on the suggestion type(s) passed in.
+
+Format: `suggest by/SUGGESTION_TYPE [by/SUGGESTION_TYPE]`
+
+* SUGGESTION_TYPE must be one of the following: `frequency`, `available` or `contract`
+
+Examples:
+
+* `suggest by/available` Obtains a list of clients where the time is 1800-2200 in the client's timezone (off work hours).
+* `suggest by/frequency` Obtains a list of clients based on the last time their details were edited in TBM. Clients who have not been contacted for a longer period will be the first in the list.
+* `suggest by/contract` Obtains a list of clients based on their current contract details. Clients whose contracts are expiring will be shown first.
+* `suggest by/contract by/available` Similar to `suggestion by/priority` but only available clients will be shown.
```

Comment:
Do you mean "Similar to `suggestion by/contract`" ?

Reply:
yea forgot to change it when i updated to mingchong's changes :(